fe2-android/app/src/main/res/layout/roll_call_fragment.xml

```diff
@@ -133,25 +133,131 @@
       app:layout_constraintStart_toStartOf="@id/roll_call_start_time"
       app:layout_constraintTop_toTopOf="@id/end_time_icon" />
 
+    <androidx.constraintlayout.widget.ConstraintLayout
+      android:id="@+id/roll_call_metadata_container"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginHorizontal="@dimen/roll_call_container_margin_horizontal"
+      android:layout_marginTop="@dimen/roll_call_container_margin_top"
+      app:layout_constraintTop_toBottomOf="@id/end_time_icon">
+
+      <com.google.android.material.card.MaterialCardView
+        android:id="@+id/roll_call_location_card"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:clickable="true"
+        app:cardCornerRadius="@dimen/roll_call_card_corner_radius"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:paddingVertical="@dimen/padding_standard">
+
+          <ImageView
+            android:id="@+id/roll_call_location_arrow"
+            android:layout_width="@dimen/roll_call_arrow_side"
+            android:layout_height="@dimen/roll_call_arrow_side"
+            android:layout_marginEnd="@dimen/roll_call_card_lateral_margin"
```

Comment:
Would it make sense to reuse some margin values of other displayed expand arrows ? I'm thinking about the ones either in the `EventListFragment` or the ones in the `DigitalCashHistoryFragment` (Though those are defined in `history_transaction_layout.xml`)
The idea is to keep as coherent as a possible a UI throughout the app (though you've probably noticed that it is very much a work in progress on this app 😄)

Reply:
Yeah it would be probably much better. I'll unify those values 👍🏼